### PR TITLE
Fixes issue #98

### PR DESCRIPTION
**Pull request by FinalAngel**
- adds tags to FAQ entries
- fixes "answer" naming

This is on hold until @cyberstar can have a look on the migrations

### CODE REVIEW

File: aldryn_faq/south_migrations/0022_add_taggit.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
```
Comment (Chive) [inline aldryn_faq/south_migrations/0022_add_taggit.py]: empty migration?

Comment (FinalAngel) [inline aldryn_faq/south_migrations/0022_add_taggit.py]: @Chive waiting for @cyberstar to step in as the django 1.6 migration show no changes using `python src/manage.py schemamigration aldryn_faq -auto`. Probably due to south not recognising the TagManager addition.

File: aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/includes/question_list.html [hunk 2]
```diff
@@ -15,6 +17,9 @@ <h3 class="panel-title">{{ qgroup.grouper }}</h3>
                 <a href="{{ question.get_absolute_url }}" class="list-group-item">
                     {% render_model question "title" %}
                     {% if not list %}<span class="badge">{{ qgroup.grouper }}</span>{% endif %}
+                    {% for tag in question.tags.all %}
+                        <span class="badge badge-default pull-right">{{ tag }}</span>
```
Comment (vxsx) [inline aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/includes/question_list.html]: you don't need `pull-right` afair
and badge-default doesn't exist

File: aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/includes/ungrouped_question_list.html [hunk 3]
```diff
@@ -5,6 +5,9 @@
     {% for question in object_list %}
         <a href="{{ question.get_absolute_url }}" class="list-group-item">
             {% render_model question "title" %}
+            {% for tag in question.tags.all %}
+                <span class="badge badge-default pull-right">{{ tag }}</span>
```
Comment (vxsx) [inline aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/includes/ungrouped_question_list.html]: same

File: aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/question_detail.html [hunk 4]
```diff
@@ -9,6 +9,14 @@
             </a>
         {% endblock faq_detail_categories %}
 
+        {% if  object.tags.all %}
+            <div class="tags pull-right">
+                {% for tag in object.tags.all %}
+                    <span class="badge badge-default">{{ tag }}</span>
```
Comment (vxsx) [inline aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/question_detail.html]: same

File: aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/question_detail.html [hunk 5]
```diff
@@ -9,6 +9,14 @@
             </a>
         {% endblock faq_detail_categories %}
 
+        {% if  object.tags.all %}
```
Comment (jmelett) [inline aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/question_detail.html]: 2 spaces

**Comment by FinalAngel** [comment]
any progress here @mkoistinen ?

**Comment by mkoistinen** [comment]
Rebased against current "master"

**Comment by FinalAngel** [comment]
NICE!

**Comment by FinalAngel** [comment]
LGTM can be merged

**Pull request merged**